### PR TITLE
Update for Navigation v0.23.0

### PR DESCRIPTION
**Pull request by danesfeder**
Refs https://github.com/mapbox/mapbox-navigation-android/issues/1504

### CODE REVIEW

File: src/pages/navigation/overview/navigation-ui.md [hunk 1]
```diff
@@ -869,7 +899,7 @@ camera.updateCameraTrackingMode(NavigationCamera.NAVIGATION_TRACKING_MODE_NORTH)
 />
 }}
 
-`NavigationCamera#showRouteOverview(int[] padding)` will also adjust the camera to the bounds of the `DirectionsRoute` being traveled along with the given padding that is passed.  
+`NavigationCamera#showRouteOverview(int[] padding)` will also adjust the camera to the bounds of the `DirectionsRoute` being traveled along with the given padding that is passed.
```
Comment (langsmith) [inline src/pages/navigation/overview/navigation-ui.md]: that is passed –> which is passed

File: src/pages/navigation/overview/route-progress.md [hunk 2]
```diff
@@ -50,6 +50,8 @@ override fun onProgressChange(location: Location, routeProgress: RouteProgress)
 | remainingWaypoints          | Number of waypoints remaining on the current route.    |
 | currentStepPoints           | List of `Point`s representing the current step geometry.  |
 | upcomingStepPoints          | List of `Point`s representing the upcoming step geometry.    |
+| voiceInstruction            | The current `VoiceInstruction` for the given segment along the route.    |
+| currentState                | `RouteStateProgress` represents the current state of route initialization and location tracking along the route.    |
```
Comment (langsmith) [inline src/pages/navigation/overview/route-progress.md]: period after `location`

Comment (langsmith) [reply src/pages/navigation/overview/route-progress.md]: whoops, nevermind

**Comment by danesfeder** [comment]
@langsmith addressed, thank you!

**Pull request merged**